Tests/eppo/EppoClientDataTests.swift

```diff
@@ -0,0 +1,144 @@
+import XCTest
+
+import Foundation
+import OHHTTPStubs
+import OHHTTPStubsSwift
+
+@testable import eppo_flagging
+
+
+struct TestSubject : Decodable {
+    let subjectKey: String;
+    let subjectAttributes: SubjectAttributes;
+    let assignment: EppoValue;
+}
+
+struct AssignmentTestCase : Decodable {
+    var flag: String = "";
+    var variationType: UFC_VariationType;
+    var defaultValue: EppoValue;
+    var subjects: [TestSubject];
+}
+
+final class EppoClientDataTests: XCTestCase {
+    var loggerSpy: AssignmentLoggerSpy!
+    var eppoClient: EppoClient!
+      
+    func testAllAssignments() async throws {
+        try await testAssignments(obfuscated: false)
+        try await testAssignments(obfuscated: true)
```

Comment:
🔥

Reply:
Minor but consider separating the tests out into two test functions/cases: one not obfuscated and one that is.